sections/collection-list.liquid

```diff
@@ -42,14 +50,14 @@
           endif
         -%}
         {%- for block in section.blocks -%}
-          <li id="Slide-{{ section.id }}-{{ forloop.index }}" class="collection-list__item grid__item{% if section.settings.swipe_on_mobile %} slider__slide{% endif %}{% if block.settings.collection.featured_image == nil %} collection-list__item--no-media{% endif %}" {{ block.shopify_attributes }}>
+          <li id="Slide-{{ section.id }}-{{ forloop.index }}" class="collection-list__item grid__item{% if show_mobile_slider %} slider__slide{% endif %}{% if block.settings.collection.featured_image == nil %} collection-list__item--no-media{% endif %}" {{ block.shopify_attributes }}>
             {% render 'card-collection', card_collection: block.settings.collection , media_aspect_ratio: section.settings.image_ratio, columns: columns %}
           </li>
         {%- endfor -%}
       </ul>
   
-      {%- if section.settings.swipe_on_mobile -%}
-        <div class="slider-buttons no-js-hidden{% if section.blocks.size < 5 %} medium-hide{% endif %}{% if section.blocks.size < 2 %} small-hide{% endif %}">
+      {%- if show_mobile_slider -%}
+        <div class="slider-buttons no-js-hidden{% if show_mobile_slider == false %} medium-hide small-hide{% endif %}">
```

Comment:
Same here. If this is all wrapped in a `{%- if show_mobile_slider -%}` then `show_mobile_slider` won't ever be anything but true.

Reply:
Same as above